pkg/services/ngalert/notifier/channels/wecom_test.go

```diff
@@ -76,6 +76,33 @@ func TestWeComNotifier(t *testing.T) {
 				"msgtype": "markdown",
 			},
 			expMsgError: nil,
+		}, {
+			name: "Custom title and message with multiple alerts",
+			settings: `{
+				"url": "https://qyapi.weixin.qq.com/cgi-bin/webhook/send?key=db0e4533-aa2a-4385-8a24-2a7b6be72d20",
```

Comment:
```suggestion
				"url": "http://localhost",
```
Let's use localhost instead a "real" url here.